frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -339,26 +419,35 @@ const SubscriptionTableRow: React.FC<SubscriptionTableRowProps> = ({
         </Link>
       </TableData>
 
-      {/* Namespace */}
-      <TableData className={tableColumnClasses[1]}>
-        <ResourceLink kind="Namespace" title={namespace} name={namespace} />
+      {/* Operator Namespace */}
+      {ns === ALL_NAMESPACES_KEY ? (
+        <TableData className={tableColumnClasses[1]}>
+          <TableData className={tableColumnClasses[1]}>
+            <ResourceLink kind="Namespace" title={namespace} name={namespace} />
+          </TableData>
+        </TableData>
+      ) : null}
+
+      {/* Managed Namespaces */}
+      <TableData className={tableColumnClasses[2]}>
+        <span className="text-muted">None</span>
       </TableData>
 
       {/* Status */}
       <TableData className={tableColumnClasses[3]}>{getStatus()}</TableData>
 
-      {/* Deployment */}
-      <TableData className={tableColumnClasses[2]}>
-        <span className="text-muted">None</span>
+      {/* Last Updated */}
+      <TableData className={tableColumnClasses[4]}>
+        {obj.status && fromNow(obj.status.lastUpdated)}
```

Comment:
You need a fallback value like `-` if `lastUpdated` isn't set in the resource.

Reply:
Hey @spadgett I've noticed in the UI we often just leave the field blank. For the Last Updated field, as a user if this info was not provided I would expect to see something more like "not available" or as we handle it now, blank. Is there a reason that you would opt for the dash over these?

Reply:
Whatever our preferred default is, we could likely create a simple component that wraps and displays `value || default`.  Eliminate some review maintenance.

Reply:
Usually we show `-` or occassionally some muted text like `No <blank>`. I can't think of a place where we leave the table cell blank (but I'm sure we do somewhere).